vhdl_lang/src/analysis/overloaded.rs

```diff
@@ -449,7 +449,7 @@ mod tests {
                 &fcall.pos,
                 &des,
                 &mut fcall.item.parameters,
-                SubprogramKind::Function(ttyp),
+                crate::analysis::overloaded::SubprogramKind::Function(ttyp),
```

Comment:
This change looks uncessary.

Reply:
It conflicts with the `ast::SubprogramKind` wich is why I had to add this